core/src/forward_packet_batches_by_accounts.rs

```diff
@@ -261,7 +241,7 @@ mod tests {
     }
 
     #[test]
```

Comment:
maybe add a test of inserting valid, invalid packets to make sure `self.accepting_packets` is functioning properly.

Reply:
added test